REST Bindings/openapi.yaml

```diff
@@ -152,9 +151,12 @@ paths:
             GS1-Extensions:
               $ref: '#/components/headers/GS1-Extensions'
             Location:
-              example: "https://example.com/capture/id9261379075"
+              description: |
+                URL of the capture job
               schema:
-                $ref: '#/components/schemas/Location'
+                example: "https://example.com/capture/id9261379075"
+                type: string
+                format: uri
```

Comment:
be careful, Location can be a URL reference thus not a URI and it is very likely it will be, for instance 

```
POST /events

201 Created
Location: /events/urn:uuid:4567
```

Reply:
Hey @jmcanterafonseca-iota not sure I fully get the comment - afaik the schema only support URI as format (vs URL) and all fields containing a URL are set as `format: uri`
Having said that you made me think about one thing: they are likely to be relative URLs vs absolute so we should probably mention this.